Plandomizer.py

```diff
@@ -1114,30 +1120,24 @@ def reset(self):
                     for world in self.world_dists:
                         world.update({k: self.src_dict[k]})
 
-
-    def starting_items_from_settings(self, world_id):
+        # normalize starting items to use the dictionary format
         data = defaultdict(lambda: StarterRecord(0))
-        if isinstance(self.settings.starting_items, dict):
-            if "starting_equipment" in self.settings.distribution._settings:
+        if isinstance(self.settings.starting_items, dict) and self.settings.starting_items:
+            if self.settings.starting_equipment:
                 raise ValueError('Incompatible starting item settings. Either move starting_equipment into starting_items or make starting_items a list')
-            if "starting_songs" in self.settings.distribution._settings:
+            if self.settings.starting_songs:
```

Comment:
I changed this line and the corresponding one above because it gave issues if starting songs were set in the GUI but not plando which I didn't think it should, is that resolved?

Reply:
(You could make the argument that this should error, but even if that's the case the error message needs to be changed to what's actually happening as the message implies plando use; I changed the error activation to match the message instead of the other way around)

Reply:
The change to line 1125 ensures that these two errors no longer appear if `starting_items` is an empty dict, se they should now only appear if a plando is involved.

Reply:
Even if starting equipment/songs is set in the gui but starting items is set in plando?

Reply:
Ah, I hadn't thought of that case. Should plando override all 3 settings in that case or should the equipment and songs from the GUI be merged in?

Reply:
I had it merging them in

Reply:
I changed it to allow `starting_items` as a dict to be combined with `starting_equipment` and/or `starting_songs` as a list, which should fix this.